ddtrace/utils/importlib.py

```diff
@@ -15,19 +22,23 @@ def __init__(self, modules):
                 self._missing_modules.append(module)
 
     def __enter__(self):
+        # type: () -> List[str]
         return self._missing_modules
 
     def __exit__(self, exc_type, exc_value, traceback):
-        return False
+        # type: (Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]) -> None
+        return
```

Comment:
Just curious, why did you remove the `return False` / what effect does this have?

Reply:
`mypy` wanted me to use `Literal` which lives in `mypy_extensions`. Alternatively it suggested changing the return type to `None`. I've opted for the latter to avoid a dependency on `mypy_extensions`